### PR TITLE
parse and evaluate elif condition statements

### PR DESCRIPTION
**Pull request by rakeshkky**
Adding `elif` (inspired from [python](https://docs.python.org/3/reference/compound_stmts.html#elif)), an `else if` conditional statement to existing `if ... else ... end` conditional syntax.
Example: The following template
```
{{ if ($.session.x-hasura-read-no-stale == "True") && ($.query_type == "query") }}
   {{$.primary_connection}}
{{ elif ($.query_type == "query") }}
   {{$.read_replicas}}
{{ else }}
   {{$.default_behaviour}}
{{ end }}
```
when provided with the following json context
```json
{
    "session": {
        "x-hasura-read-no-stale": "False"
    },
    "query_type": "query",
    "primary_connection": "USE_PRIMARY_CONNECTION",
    "default_behaviour": "DEFAULT_CONNECTION_BEHAVIOUR",
    "read_replicas": "USE_READ_REPLICAS"
}
```
resolves to
```
"USE_READ_REPLICAS"
```

### CODE REVIEW

File: grammar.bnf [hunk 1]
```diff
@@ -59,7 +59,14 @@ mident
...
 iff
-  : '{{' 'if' expr '}}' expr '{{' 'else' '}}' expr '{{' 'end' '}}'
+  : '{{' 'if' expr '}}' expr elif_exprs '{{' 'else' '}}' expr '{{' 'end' '}}'
+
+elif_exprs
+  : {- empty -}
+  | elif_exprs elif_expr
+
+elif_exp
+  : '{{' 'elif' expr '}}' expr
```
Comment (solomon-b) [inline grammar.bnf:69]: The grammar.bnf file is actually more out of date. Can you run this command from the root of the repo and commit the change?
```
happy src/Kriti/Parser/Grammar.y --pretty=grammar.bnf
```

File: test/data/parser/success/examples/ifStatements.kriti [hunk 2]
```diff
@@ -13,5 +13,6 @@
   {{ if ((1) <= 3) }} true {{ else }} false {{ end }},
   {{ if "foo" in ["foo"] }} 1 {{ else }} 0 {{ end }},
   {{ if "foo" in { "foo": true } }} 1 {{ else }} 0 {{ end }},
-  {{ if ("foo" in { "foo": true }) }} 1 {{ else }} 0 {{ end }}
+  {{ if ("foo" in { "foo": true }) }} 1 {{ else }} 0 {{ end }},
+  {{ if ("foo" in { "foo": false }) }} 1 {{ elif ("foo" in { "foo": true }) }} 0 {{ else }} 0 {{ end }}
```
Comment (solomon-b) [inline test/data/parser/success/examples/ifStatements.kriti:17]: can you add an example with multiple elifs in an if statement?

**Review by solomon-b — COMMENTED** [review]
Thanks for the PR! 

Everything looks good, I just have 2 small requests.

**Comment by rakeshkky** [comment]
@solomon-b I addressed your review suggestions in the latest commits.

**Review by solomon-b — APPROVED** [review]
Looks good, thank you!

**Comment by solomon-b** [comment]
Oh sorry, one last thing we forgot about is the readme. Can you add an elif example to `README.md`?

**Pull request merged**